R/derive_var_studyeye.R

```diff
@@ -0,0 +1,61 @@
+#' Derive Study Eye
+#'
+#' Derive Study Eye (`STUDYEYE`) in the ADSL dataset
+#'
+#' @param dataset_adsl ADSL input dataset
+#' @param dataset_sc SC input dataset
+#'
+#' @details
+#' Study Eye is derived in ADSL using the "Study Eye selection" records
+#' in the SC SDTM dataset.
+#'
+#' @author Edoardo Mancini
+#'
+#' @return The input ADSL dataset with an additional column named `STUDYEYE`
+#' @keywords adsl derivation ophthalmology
+#' @export
+#'
+#' @examples
+#' adsl <- tibble::tribble(
```

Comment:
Can you put `library(tibble)` and then just have `adsl <- tribble`. This is new gudance we are following. Want to avoid `packagename::function`

Reply:
Thanks, fixed.